package.json

```diff
@@ -68,7 +68,7 @@
     "knex": "0.x"
   },
   "devDependencies": {
-    "@types/knex": "^0.14.5",
+    "@babel/polyfill": "^7.2.5",
```

Comment:
Knex throws an error of polyfill dependency is not available now since it relies on it for Node 6 support since 0.16.x

Reply:
Crap! We can only remove node 6 support in 2.0 since we follow semver. We'll just have to live with this dependency.

Reply:
Oh, it's a devDependency. Well that's ok I quess. Do all knex users need to install that?

Reply:
The knex `lib` is littered with these `var _assign = require('babel-runtime/core-js/object/assign');` so I quess everyone needs to always install that. Why isn't it just a dependency?

Reply:
Never mind. I just hadn't built knex in a while :man_facepalming:

Reply:
>Do all knex users need to install that?

Only those who are still using Node 6. But they get pretty explicit error message if they don't, so hopefully it's an easy fix on their side. It is sorta kinda somewhat breaking change, but Node 6 is getting deprecated soon anyway.